extension-base/src/main/java/com/azure/autorest/extension/base/model/codemodel/Response.java

```diff
@@ -11,6 +11,7 @@
 
     private Schema schema;
 
+    private Boolean binary;
```

Comment:
We are mixing SchemaResponse and BinaryResponse here. Do we need to separate the two?

Reply:
I cannot figure out polymorphic deserialization in snakeyaml. I'm fine with this.